### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by Fantom409**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: src/OAuth1.php [hunk 1]
```diff
@@ -78,12 +78,6 @@ public function buildAuthUrl(
         array $params = []
     ): string {
         $requestToken = $this->fetchRequestToken($incomingRequest);
-        if (!is_object($requestToken)) {
```
Comment (samdark) [inline src/OAuth1.php:81]: Why was it removed?

Comment (Fantom409) [reply src/OAuth1.php:81]: It was moved into fetchRequestToken, because this method can only return OAuthToken

Comment (samdark) [reply src/OAuth1.php:81]: I don't see `$this->getState('requestToken');` in `fetchRequestToken`.

**Comment by samdark** [comment]
Closing since it can't be applied anymore.

**Pull request closed without merging**